src/modules/facility-activation-transaction/dto/get-facility-activation-transaction-response.dto.ts

```diff
@@ -0,0 +1,130 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { ENUMS, EXAMPLES, PROPERTIES } from '@ukef/constants';
+import { DateOnlyString } from '@ukef/helpers';
+
+export class GetFacilityActivationTransactionResponseDto {
+  @ApiProperty({
+    description: 'The identifier of the portfolio in ACBS. The UKEF portfolio is E1.',
+    minLength: 0,
+    maxLength: 2,
+    example: PROPERTIES.GLOBAL.portfolioIdentifier,
+  })
+  readonly portfolioIdentifier: string;
+
+  @ApiProperty({
+    description: 'The identifier of the facility in ACBS.',
+    minLength: 10,
+    maxLength: 10,
+    example: EXAMPLES.FACILITY_ID,
+  })
+  readonly facilityIdentifier: string;
+
+  @ApiProperty({
+    description: 'A numeric code denoting the status of the bundle.',
+    minLength: 0,
+    maxLength: 2,
+    example: EXAMPLES.BUNDLE_STATUS_CODE,
+  })
+  readonly bundleStatusCode: string; // TODO APIM-125: should this use ENUMS.BUNDLE_STATUSES?
```

Comment:
@avaitonis

Reply:
Maybe new Enum with these values can be useful.
10 - Pending Input With Drafts
15 - Pending Input
20 - Pending Approval
25 - Approved
30 - Denied
35 - Pending Posting
40 - Currenting Posting
50 - Posted
55 - Posting Error
80 - History Posted
85 - History Cancelled
90 - Cancelled

Reply:
Added new enum and used it in DTO